dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -35,7 +35,6 @@ commons-compiler/3.1.4//commons-compiler-3.1.4.jar
 commons-compress/1.20//commons-compress-1.20.jar
 commons-crypto/1.1.0//commons-crypto-1.1.0.jar
 commons-dbcp/1.4//commons-dbcp-1.4.jar
-commons-httpclient/3.1//commons-httpclient-3.1.jar
```

Comment:
Apache Spark has this at least since Apache Spark 1.5.x.
- https://github.com/apache/spark/blob/branch-1.5/dev/deps/spark-deps-hadoop-2.6#L44